doc/nrf/known_issues.rst

```diff
@@ -845,42 +845,42 @@ KRKNWK-9214: Pigweed submodule may not be accessible from some regions
 HomeKit
 =======
 
```

Comment:
To be included:

KRKNWK-14081 HomeKit SDK light bulb example doesn't work with MTD=y  [KRKNWK-14081](https://projecttools.nordicsemi.no/jira/browse/KRKNWK-14081) - HomeKit SDK light bulb example doesn't work with MTD=y 
OPEN v2.0.0-1v1.9.1v1.9.0v1.8.0v1.7.1v1.7.0v1.6.1v1.6.0

KRKNWK-13249 Unexpected assertion at line 197 in HAPBLEPeripheralManager.c  [KRKNWK-13249](https://projecttools.nordicsemi.no/jira/browse/KRKNWK-13249) - Unexpected assertion at line 197 in HAPBLEPeripheralManager.c 
IN PROGRESS v2.0.0-1v1.9.1v1.9.0

KRKNWK-13607 Stateless switch application crashes upon factory reset  [KRKNWK-13607](https://projecttools.nordicsemi.no/jira/browse/KRKNWK-13607) - Stateless switch application crashes upon factory reset 
OPEN v2.0.0

KRKNWK-13010 Dropping from Thread to BLE takes too long  [KRKNWK-13010](https://projecttools.nordicsemi.no/jira/browse/KRKNWK-13010) - Dropping from Thread to BLE takes too long
RESOLVED v2.0.0-1v1.9.1v1.9.0v1.8.0v1.7.1v1.7.0v1.6.1v1.6.0

_Workaround:  Specification for TCT030 is going to be updated_

KRKNWK-11729 Wrong Stateless Switch Event characteristic value handling  [KRKNWK-11729](https://projecttools.nordicsemi.no/jira/browse/KRKNWK-11729) - Wrong Stateless Switch Event characteristic value handling RESOLVED v2.0.0-1v1.9.1v1.9.0v1.8.0v1.7.1v1.7.0v1.6.1v1.6.0

_Workaround:  Specification in point 11.47 is going to be updated_

Reply:
Hey @annwoj , would you be able to provide more detailed descriptions for these issues?

Reply:
Updated